### ISSUE TITLE
Motif-Only Matrix Profile (MOMP)

### ISSUE THREAD

**Issue by seanlaw** [TDAmeritrade/stumpy#1031]
Keogh's publicly announced his [final matrix profile paper](https://www.dropbox.com/scl/fi/mt8vp7mdirng04v6llx6y/MOMP_DeskTop.pdf)

This might be interesting to explore and reproduce

**Comment by joehiggi1758** [comment]
Hey @seanlaw hope you're having a wonderful day! 

Finally getting some time to look into this! Should the final output be a `.ipynb` notebook?

**Comment by seanlaw** [comment]
Yes, a .ipynb is the final output to be created and reviewed but not necessarily saved into the repo

**Comment by NimaSarajpoor** [comment]
@joehiggi1758 
> are there any previous ones I can mirror?

Examples:
* [Discord Merlin](https://github.com/TDAmeritrade/stumpy/blob/main/docs/WIP/Tutorial_DiscordMERLIN.ipynb)
* [Time Series Chain](https://github.com/TDAmeritrade/stumpy/blob/main/docs/WIP/Tutorial_Geometric_Chains.ipynb)

For the sake of consistency, try to follow the naming conventions in STUMPY. For instance, if possible, use `T` (or `T_A`) for a time series, and `m` for the window size (Maybe also take a look at the docstring of the function `stump` in `stumpy/stump.py`). In addition, try to push changes in small chunks. This can help others to follow your work/changes better, and provide better feedback.
